go/vt/vtgate/planbuilder/testdata/select_cases.json

```diff
@@ -5170,7 +5170,7 @@
         "Vindex": "user_index"
       },
       "TablesUsed": [
-        "user.dual",
+        "main.dual",
         "user.user"
       ]
```

Comment:
this change looks incorrect to me

Reply:
this is giving wrong information, though the plan is correct.

Reply:
Fixed, no longer any diff.